pkg/cmd/cli/cmd/expose.go

```diff
@@ -56,7 +56,13 @@ func NewCmdExpose(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.C
 		cmdutil.CheckErr(err)
 		defRun(cmd, args)
 	}
-	cmd.Flags().String("hostname", "", "Set a hostname for the new route")
+	cmd.Flags().String("hostname", "", "Set a hostname for the new route. Optional.")
+	cmd.Flags().String("path", "", "Path that the router watches to route traffic to the service. Optional.")
+	cmd.Flags().String("tls-termination", "", "Indicate a termination type for the generated route. If not set, any termination config will be ignored. Supports 'edge', 'passthrough', or 'reencrypt'.")
```

Comment:
I can validate inside the generator and in [our validation step](https://github.com/openshift/origin/blob/fcfd4707b41a362c10dd29c541cb1e764cccae7f/pkg/cmd/cli/cmd/expose.go#L65), this is not a problem. Unfortunately we cannot follow the complete, validate, run pattern here since we re-use expose from upstream and ... they don't follow it. @fabianofranz fyi see [oc logs](https://github.com/openshift/origin/blob/fcfd4707b41a362c10dd29c541cb1e764cccae7f/pkg/cmd/cli/cmd/logs.go#L87) which follows the pattern both upstream and in Origin.